web/public/locales/en/views/system.json

```diff
@@ -91,6 +91,11 @@
       "tips": "This value represents the total storage used by the recordings in Frigate's database. Frigate does not track storage usage for all files on your disk.",
       "earliestRecording": "Earliest recording available:"
     },
+    "shm": {
+      "title": "SHM (shared memory) allocation",
+      "warning": "The current SHM size of {{total}}MB is too small. Increase it to at least {{min_shm}}MB.",
+      "readTheDocumentation": "Read the documentation"
```

Comment:
Can we just have `read the documentation` part of common so we don't need to add a different variant of it to every place we use it?

Reply:
Yeah, I was wondering why that never happened originally. I'll refactor all that in a separate PR